Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM python:3.8
 
+RUN apt update && apt install -y git
```

Comment:
Formally, `apt` does not have a stable command-line CLI and will emit a warning to be careful with using it in scripts. I typically use `apt-get` instead of `apt` instead.

Reply:
That's a good thing to note. I think we'll end up refactoring this Dockerfile in future to something more production suitable, so that can be included.